### PR TITLE
Corrects incorrect link to GraphIQL Headers documentation in Playground comment

### PR DESCRIPTION
**Pull request by alirezaRaisSattari**
in redwood graphql playground redirect to wrong link

### CODE REVIEW

File: packages/graphql-server/src/createGraphQLYoga.ts [hunk 1]
```diff
@@ -99,7 +99,7 @@ export const createGraphQLYoga = ({
   }`
 
     // TODO: Once Studio is not experimental, can remove these generateGraphiQLHeaders
-    const authHeader = `{"x-auth-comment": "See documentation: https://redwoodjs.com/docs/cli-commands#setup-graphiQL-headers on how to auto generate auth headers"}`
+    const authHeader = `{"x-auth-comment": "See documentation: https://redwoodjs.com/docs/cli-commands#setup-graphiqL-headers on how to auto generate auth headers"}`
```
Comment (dthyresson) [inline packages/graphql-server/src/createGraphQLYoga.ts:102]: I don't think your link is correct. Is it not:

https://redwoodjs.com/docs/cli-commands#setup-graphiql-headers

You have a capital "L".

**Review by dthyresson — CHANGES_REQUESTED** [review]
Could you 

* double check the url
* re title the PR to be more descriptive of the change so that the title in release notes let's people know the contribution.
Something like: "Corrects incorrect link to GraphIQL Headers documentation in Playground comment" or something along those lines

File: packages/graphql-server/src/createGraphQLYoga.ts (hunk 1 above)
Comment (alirezaRaisSattari) [reply packages/graphql-server/src/createGraphQLYoga.ts]: yes I fix that

**Review by dthyresson — COMMENTED** [review]
Approved

**Pull request merged**